src/main/java/org/apache/sling/distribution/DistributionRequest.java

```diff
@@ -18,15 +18,17 @@
  */
 package org.apache.sling.distribution;
 
-import aQute.bnd.annotation.ProviderType;
+import javax.annotation.Nonnull;
+import javax.annotation.ParametersAreNonnullByDefault;
 
-import org.jetbrains.annotations.NotNull;
+import org.osgi.annotation.versioning.ProviderType;
 
 /**
  * A {@link org.apache.sling.distribution.DistributionRequest} represents the need from the caller to have
  * some content being distributed from a source instance to a target instance.
  */
 @ProviderType
+@ParametersAreNonnullByDefault
```

Comment:
@cschneider How about moving this to the package-info.java and mark all classes at once ?

Reply:
Good idea